src/razor/src/document/razorDocumentSynchronizer.ts

```diff
@@ -170,9 +169,8 @@ export class RazorDocumentSynchronizer {
                 }
             },
             dispose: () => {
-                while (rejectionsForCancel.length > 0) {
-                    rejectionsForCancel.pop();
-                }
+                rejectionsForCancel.length = 0;
```

Comment:
Looked it up and apparently setting length to 0 and using slice to remove all the elements are the fastest ways to clear an array. #TheMoreYouKnow